src/Goerli-DssSpell.t.sol

```diff
@@ -518,166 +520,120 @@ contract DssSpellTest is GoerliDssSpellTestBase {
-        vote(address(cureSpell_008));
+        giveAuth(address(oracle), address(this));
 
-        cureSpell_008.schedule();
-        uint256 castTime = block.timestamp + pause.delay();
-        hevm.warp(castTime);
-        cureSpell_008.cast();
+        oracle.cure("RWA008-A");
     }
 
-    function testSpellIsCast_RWA008_INTEGRATION_TELL_CULL() public {
-        if (!spell.done()) {
-            vote(address(spell));
-            scheduleWaitAndCast(address(spell));
-            assertTrue(spell.done());
-        }
-        assertTrue(oracle.good("RWA008-A"));
+    function testRWA008_INTEGRATION_TELL_CULL() public {
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
 
-        tellSpell_008 = new TellSpellRwa008();
-        vote(address(tellSpell_008));
+        giveAuth(address(vat), address(this));
+        giveAuth(address(oracle), address(this));
 
-        tellSpell_008.schedule();
+        assertTrue(oracle.good("RWA008-A"));
+
+        vat.file("RWA008-A", "line", 0);
+        oracle.tell("RWA008-A");
 
-        uint256 castTime = block.timestamp + pause.delay();
-        hevm.warp(castTime);
-        tellSpell_008.cast();
         assertTrue(!oracle.good("RWA008-A"));
 
-        cullSpell_008 = new CullSpellRwa008();
-        vote(address(cullSpell_008));
+        oracle.cull("RWA008-A", addr.addr("RWA008_A_URN"));
 
-        cullSpell_008.schedule();
-        castTime = block.timestamp + pause.delay();
-        hevm.warp(castTime);
-        cullSpell_008.cast();
         assertTrue(!oracle.good("RWA008-A"));
         (, address pip, , ) = oracle.ilks("RWA008-A");
         assertEq(DSValueAbstract(pip).read(), bytes32(0));
     }
 
-    function testSpellIsCast_RWA008_OPERATOR_GET_RWA008_TOKEN() public {
-        assertEq(GemAbstract(address(rwagem_008)).balanceOf(rwaOperator_008), 1 * WAD);
+    function testRWA008_OPERATOR_GET_RWA008_TOKEN() public {
+        assertEq(rwagem_008.balanceOf(rwaOperator_008), 1 * WAD);
     }
 
-    function testSpellIsCast_RWA008_OPERATOR_LOCK_DRAW_CONDUITS_WIPE_FREE() public {
-        if (!spell.done()) {
-            vote(address(spell));
-            scheduleWaitAndCast(address(spell));
-            assertTrue(spell.done());
-        }
+    function testRWA008_OPERATOR_LOCK_DRAW_CONDUITS_WIPE_FREE() public {
+        vote(address(spell));
+        scheduleWaitAndCast(address(spell));
+        assertTrue(spell.done());
 
         // TODO: Figure out why warping here makes this test fail
         // hevm.warp(block.timestamp + 10 days); // Let rate be > 1
 
         // set the balance of this contract
-        hevm.store(address(rwagem_008), keccak256(abi.encode(address(this), uint256(3))), bytes32(uint256(2 * WAD)));
+        hevm.store(address(rwagem_008), keccak256(abi.encode(address(this), uint256(3))), bytes32(uint256(1 * WAD)));
+        assertEq(rwagem_008.balanceOf(address(this)), 1 * WAD);
```

Comment:
What's the point of this assertion? Just make sure that the `hevm.store` above works?

Reply:
This was already in the test I think, but yeah, it should be a sanity check.